rust-toolchain

```diff
@@ -1 +0,0 @@
-1.64.0
```

Comment:
I thought this was our way declaring the MSRV of our project and even an attempt to unify the Rust version that every developer uses, to avoid feature compatibility issues.

Reply:
The decision has been made to remove the toolchain file so that we have some diversity amongst developers to find potential issues earlier, though the CI should used a fixed version to ensure that the release artifacts are using a fixed rust version.